dsaas-services/f8-webhook.yaml

```diff
@@ -9,7 +9,3 @@ services:
     parameters:
       REPLICAS: 2
       IMAGE: quay.io/openshiftio/rhel-fabric8-services-fabric8-webhook
-  - name: production
```

Comment:
@khrm can you elaborate why you have removed this?

Reply:
@riuvshyn Because this service is new and we want to take it to preprod to see how it behaves before taking it on prod.